staging/src/k8s.io/client-go/scale/scheme/validation/validation.go

```diff
@@ -0,0 +1,36 @@
+/*
+Copyright 2017 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package validation
+
+import (
+	"k8s.io/apimachinery/pkg/util/validation/field"
+	apivalidation "k8s.io/kubernetes/pkg/api/validation"
+
+	"k8s.io/client-go/scale/scheme"
+)
+
+// ValidateScale is used to validate scheme.Scale
+func ValidateScale(scale *scheme.Scale) field.ErrorList {
```

Comment:
I don't see what purpose this serves.  This is never supposed to be used as an API server type.

Reply:
Removed.